cypress/support/commands.ts

```diff
@@ -126,6 +141,38 @@ Cypress.Commands.add('mockAllRequests', () => {
   cy.mockInternalRequests()
 })
 
+// @ts-ignore
+Cypress.Commands.add('mockWebSocketRequests', (method: string, data: Object, response: Object) => {
+  let options = {
+    takeWhileFn: (message: IMessage) => message && message.method !== "end",
+    startUpMessage: {
+      method: method,
+      data: data
+    },
+  };
+  // Wrap the request in order to bypass the defaultCommandTimeout
+  // Investigating alternative solutions
+  cy.wrap(null, { timeout: 10000 }).then(() =>
+    cy.streamRequest(wsConfig, options).then(results => {
+      const connectionResult = results && results[0];
+      expect(connectionResult).to.not.be.undefined;
+      expect(connectionResult).to.have.property("method", "connect");
+      expect(connectionResult).to.have.property("data", "connect success");
```

Comment:
These break our lint rules because `eslint` doesn't understand that these are getters with side effects.

Probably best to disable `no-unused-expressions` linting for these lines.